catalyst_voices/lib/pages/registration/upload_seedphrase_confirmation_dialog.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:catalyst_voices/widgets/avatars/voices_avatar.dart';
+import 'package:catalyst_voices/widgets/buttons/voices_filled_button.dart';
+import 'package:catalyst_voices/widgets/buttons/voices_text_button.dart';
+import 'package:catalyst_voices/widgets/modals/voices_alert_dialog.dart';
+import 'package:catalyst_voices/widgets/modals/voices_dialog.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:flutter/material.dart';
+
+class UploadSeedphraseConfirmationDialog extends StatelessWidget {
```

Comment:
`seed phrase` is two separate words so we should capitalize it like this:

```suggestion
class UploadSeedPhraseConfirmationDialog extends StatelessWidget {
```